Assignment2/philippScomparin-PhilippScomparin.ttl

```diff
@@ -0,0 +1,10 @@
+@prefix vocab: <http://example-rdf.org/properties/> .
+@base <http://example.org/> .
+
+<Class01> vocab:includes <Sensor029> ;
+vocab:includes <Computer101> .
+<Sensor029> vocab:hasMeasurement <Measurement8401> .
+<Measurement8401> vocab:hasTemperature "29" ;
```

Comment:
You should add datatypes to integers and dates

Reply:
Ok, I have added them now